### ISSUE TITLE
Form Item helper and error slots not accessible

### ISSUE THREAD

**Issue by dustin-nielsen-goa** [GovAlta/ui-components#2578]
# Info

Found during the fix for #2365 . Error and helper text when a slot (ie. using html instead of just a string) are not read by screen reader.

## Acceptance Criteria:

1. Error and helper text when used as a slot (html) should be read by a screen reader as it would be read if it was just a string.
